### PR TITLE
feat(ark, arkctapp): dynamic backscraper

### PR DESCRIPTION
**Pull request by grossir**
Solves #958

Also, implements scraper on new HTML source page,instead of using RSS feed

### CODE REVIEW

**Review by quevon24 — CHANGES_REQUESTED** [review]
I was able to find a false positive item, i think we should take that into account:

```
Adding new item:
    case_dates: 2019-06-26
    case_names: "NO OPINIONS THIS WEEK"
    download_urls: "https://opinions.arcourts.gov/ark/courtofappeals/en/417492/1/document.do"
    precedential_statuses: "Published"
    blocked_statuses: False
    date_filed_is_approximate: False
    docket_numbers: ""
    citations: []
    case_name_shorts: ""

```

Also, when i tried to run the scrapers for the first time i got a 403 error due to a captcha, i guess if this happens we only need to rerun the scraper until it works, or juriscraper has something to solve the captchas?

image

After running them a second time they worked without problems.

**Comment by flooie** [comment]
Why did you drop the RSS feed?

**Comment by grossir** [comment]
We can't backcrape RSS feeds; if we kept the RSS scraper we would need 2 scraper files, which I find messy. Also, the RSS scraper may be a potential cause for the gaps? It had this comment`""This scraper has an intential 10 opinion limit to avoid lexum cap."""`

**Comment by flooie** [comment]
I have one overarching concern here - in testing a few things - I have triggered a captcha- and I think we have likely not run into this yet because we have been using the RSS feed.  @grossir - have you hit a captcha yet at all?

**Comment by flooie** [comment]
okay, I think I may have just been over eager here.  Let's merge it and see if we get blocked by Lexus - who we have worked with in the past.

**Comment by grossir** [comment]
There were a bunch of invalid opinion strings, thanks for catching that Kevin
image

File: tests/local/test_ScraperExtractFromTextTest.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ class ScraperExtractFromText(unittest.TestCase):
             # Some opinions don't have dockets because Arkansas publishes important announcements.
             (
                 """Cite as 2022 Ark. 14\nSUPREME COURT OF ARKANSAS Opinion Delivered: January 27, 2022""",
-                {"OpinionCluster": {"docket_number": ""}},
+                {},
```
Comment (flooie) [inline tests/local/test_ScraperExtractFromTextTest.py:54]: I dont understand what is going on here

File: tests/examples/opinions/united_states/arkctapp_example.compare.json [hunk 2]
```diff
@@ -1,112 +1,170 @@
...
+    "citations": "2024 Ark. App. 226",
+    "case_name_shorts": "",
+    "per_curiam": false
   },
   {
-    "case_dates": "2022-02-02",
-    "case_names": "Fred Jackson v. Director, Department of Workforce Services",
-    "download_urls": "https://opinions.arcourts.gov/ark/courtofappeals/en/520510/1/document.do",
+    "case_dates": "2024-04-03",
+    "case_names": "JASON REYNOLDS v. MARY \"KATIE\" REYNOLDS",
```
Comment (flooie) [inline tests/examples/opinions/united_states/arkctapp_example.compare.json:88]: and can we improve the case names from all caps?

File: juriscraper/opinions/united_states/state/ark.py [hunk 3]
```diff
@@ -2,60 +2,103 @@
...
+        params = {
+            "cont": "",
+            "ref": "",
+            "d1": start.strftime("%m/%d/%Y"),
+            "d2": end.strftime("%m/%d/%Y"),
+            "col": self.court_code,
+            "tf1": "",
+            "tf2": "",
+            "or": "date",
+            "iframe": "true",
```
Comment (flooie) [inline juriscraper/opinions/united_states/state/ark.py:98]: do we need all of these empty fields or can we remove them.

Comment (grossir) [reply juriscraper/opinions/united_states/state/ark.py:98]: I think without them it stops working

Just commented them out and the run `arkctapp` normal scraper 
- Without: 0 results
- With: 1 result (Skipping 07-17-24 SYLLABUS)

**Review by flooie — APPROVED** [review]

**Pull request merged**